core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java

```diff
@@ -74,7 +74,7 @@ public Response searchRecentDocuments(@ApiParam(value = "Query string", required
     if (myWork) {
       recentFilters.add(filterMyWorkingDocuments());
     }
-    recentFilters.add(getFileTypesFilter());
+    recentFilters.add(getFileTypesFilter(myWork));
     Identity currentIdentity = ConversationState.getCurrent().getIdentity();
     UserACL userACL = PortalContainer.getInstance().getComponentInstanceOfType(UserACL.class);
     if (!currentIdentity.isMemberOf(userACL.getAdminGroups(), "*")) {
```

Comment:
Could you change this to : 
```suggestion
    if ( !userACL.isSuperUser() && !userACL.isUserInGroup(userACL.getAdminGroups()))) {
```

Reply:
Updated !